DigitalLearningSolutions.Web/Styles/learningMenu/index.scss

```diff
@@ -1,7 +1,13 @@
-﻿.incomplete {
-  border-left: 6px solid #d5281b;
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+.section-card {
```

Comment:
I don't think this is necessarily a very good name, as we're likely to use it for all cards not just sections. Maybe `learning-menu-card`? 🙂